### DISCUSSION TITLE
可以选择开关PCL2动画

### DISCUSSION THREAD

**Discussion by rong6** [Meloong-Git/PCL#1349] [提问] [answered]
虽然但是这对我没啥用，不过还是建议整个可以开关PCL2动画，防止某些渣机卡成shift（~~虽然但是不可能连PCL2都带不动吧~~），也可以整个流畅模式之类的东西 :-D

**Comment by WForst-Breeze** [comment, marked as answer]
请前往 **设置-启动器-调试选项-动画速度** 调整动画速度滑块至 “关闭” 即可（说人话就是，拉到最右侧）
